.github/workflows/CI.yml

```diff
@@ -14,31 +14,30 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6]
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        python-version: [ 3.7 ]
+        os: [ windows-latest, ubuntu-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
     permissions:
       id-token: write
       contents: read
-    name: Build on ${{ matrix.os }}
+    name: Build on ${{ matrix.os }} with Python ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Validate commit messages 
+      - name: Validate commit messages
         uses: wagoid/commitlint-github-action@v4
         if: matrix.os == 'ubuntu-latest'
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install CLI and verify
+      - name: Python version
         run: |
+          python -VV
           python -m pip install --upgrade pip
-          pip install .
```

Comment:
Why are we making this change when it was added as part of https://github.com/aws-greengrass/aws-greengrass-gdk-cli/pull/56?

Reply:
I moved the gdk installation closer to UAT step. The installation is not removed.

Reply:
Synced up offline. The responsibility of verifying clean install (originally identified in #56) is now part of regression job.